backends/pkg/reconciler/eventmesh/handler.go

```diff
@@ -118,3 +161,108 @@ func fetchEventTypes(eventTypeLister eventinglistersv1beta2.EventTypeLister, log
 
 	return convertedEventTypes, err
 }
+
+func buildSubscriptions(ctx context.Context, triggerLister eventinglistersv1.TriggerLister, brokerMap map[string]*Broker, etNameMap map[string]*EventType, logger *zap.SugaredLogger) (*SubscriptionMap, error) {
+	// map key: "<namespace>/<eventType.spec.type>"
+	subscriptionMap := make(SubscriptionMap)
+
+	dynamicClient := dynamicclient.Get(ctx)
+
+	triggers, err := triggerLister.List(labels.Everything())
+	if err != nil {
+		logger.Errorw("Error listing triggers", "error", err)
+		return &subscriptionMap, err
+	}
+
+	for _, trigger := range triggers {
+		// if the trigger's broker is not set or if we haven't processed the broker, we can skip the trigger
+		if trigger.Spec.Broker == "" {
+			continue
+		}
+		brokerRef := NameAndNamespace(trigger.Namespace, trigger.Spec.Broker)
+		if _, ok := brokerMap[brokerRef]; !ok {
+			continue
+		}
+
+		// if the trigger has no subscriber, we can skip it, there's no relation to show on Backstage side
+		if trigger.Spec.Subscriber.Ref == nil {
+			continue
+		}
+
+		subscriberBackstageId, err := getSubscriberBackstageId(ctx, dynamicClient, trigger, logger)
+		if err != nil {
+			// do not stop the Backstage plugin from rendering the rest of the data, e.g. because
+			// there are no permissions to get a single subscriber resource
+			continue
```

Comment:
Can either log the error or collect them and log them all together at the end ? At warning/info level ?